openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -65,53 +65,70 @@ <h1>$_('Community Edit Requests')</h1>
+                      <!-- this is hidden from display and screen readers -->
+                      <textarea rows="1" placeholder="$_('Add a comment...')"></textarea>
+                      <input class="mr-comment-btn" type="button" value="REPLY" data-mrid="$r['id']" data-merge-type="$r['mr_type']" data-username="$username">
+                    </div>
+                </div>
 
+                <span class="mr-opened">MR #$r['id'] opened $datestr(r['created']) by
+                  <a>
+                    @$r['submitter']
+                  </a>
+                </span>
               </div>
             </td>
 
             <td class="mr-comment-review-cell">
-              $if is_open:
-                $if is_submitter:
-                  <a class="mr-close-link" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="javascript:;">$_('Close')</a>
-                $elif can_merge:
-                  $ show_link = not r.get('reviewer') or r.get('reviewer') == username
-                  $ extra_classes = '' if show_link else ' hidden'
-                  <button class="mr-comment-review-cell__review">
-                    <strong>
-                      <a class="mr-resolve-link$extra_classes" id="mr-resolve-link-$r['id']" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="$url" target="_blank">$_('REVIEW')</a>
-                    </strong>
-                  </button>
-                  <!-- ADD COMMENT BUTTON HERE -->
+              <div>
+                $if can_merge and is_open and r.get('reviewer'):
+                  <div class="reviewer-section">
+                    <span class="reviewer" >@$r['reviewer']</span> <span class="mr-unassign" data-mrid="$r['id']" data-merge-type="$r['mr_type']" style="margin-left: 3px;">&times;</span>
+                  </div>
+                $elif r.get('reviewer'):
+                  <div class="reviewer-section">
+                    <img src="/static/images/icons/icon_check.png" style="height: 10px; margin-right: 3px;"><span class="reviewer">@$r['reviewer']</span>
+                  </div>
+                $else:
+                  $if is_submitter:
+                    <a class="mr-close-link" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="javascript:;">$_('Close')</a>
+                  $elif can_merge:
+                    $ show_link = not r.get('reviewer') or r.get('reviewer') == username
+                    $ extra_classes = '' if show_link else ' hidden'
+                    <button class="mr-comment-review-cell__review">
+                      <strong>
+                        <a class="mr-resolve-link$extra_classes" id="mr-resolve-link-$r['id']" data-mrid="$r['id']" data-merge-type="$r['mr_type']" href="$url" target="_blank">$_('REVIEW')</a>
+                      </strong>
+                    </button>
+              </div>
+              <div>
+                <button id="mr-comment-btn" class="mr-comment-review-cell__comments mr-comment-btn-$r['id']">
```

Comment:
Does this button need an `id`?  If so, make sure that the ID is unique.  Several of these buttons will be on a single page.

Reply:
I think it needs an id for the toggle in the backend.

Reply:
TODO: Use unique IDs on integration branch